### DISCUSSION TITLE
is milvus and manu the same?

### DISCUSSION THREAD

**Discussion by siddhsql** [milvus-io/milvus#31465] [Q&A and General discussion]
where can i find manu? https://zilliz.com/blog/anatomy-of-a-cloud-native-vector-database-management-system

**Comment by yhmo** [comment]
If you read the first paragraph of the article, you will see "Manu (project name for Milvus 2.0)", so we know Manu is the project name of Milvus 2.0. And the link is also pointing to milvus github repo.

**Comment by xiaofan-luan** [comment]
> where can i find manu? https://zilliz.com/blog/anatomy-of-a-cloud-native-vector-database-management-system

Manu is milvus 2.0, correct. Just the internal code
